code/src/test_srcs/test__sbi_get_marchid/test__sbi_get_marchid.c

```diff
@@ -0,0 +1,33 @@
+#include "test_macros.h"
```

Comment:
Are you facing any issues running spike?

Reply:
Now, it is running